File: 448-find-all-numbers-disappeared-in-an-array/find-all-numbers-disappeared-in-an-array.py
class Solution(object):
    def findDisappearedNumbers(self, nums):
        """
        :type nums: List[int]
        :rtype: List[int]
        """
        n=len(nums)
        seen=set(nums)
        ans=[] 
        for i in range(1,n+1):
            if i not in seen:
                ans.append(i)
        return ans
        

File: 448-find-all-numbers-disappeared-in-an-array/find-all-numbers-disappeared-in-an-array.java
class Solution {
    public List<Integer> findDisappearedNumbers(int[] nums) {

        int n = nums.length;

        Set<Integer> seen = new HashSet<>();

        for (int num : nums) {
            seen.add(num);
        }

        List<Integer> ans = new ArrayList<>();

        for (int i = 1; i <= n; i++) {
            if (!seen.contains(i)) {
                ans.add(i);
            }
        }

        return ans;
    }
}